### ISSUE TITLE
Some (not so) special characters get encoded

### ISSUE THREAD

**Issue by tmlmt** [harlan-zw/nuxt-og-image#32]
### Describe the bug

For instance a single quote `'` is turned into `&#39;`

![image](https://user-images.githubusercontent.com/10244927/227773228-db271380-a586-4d47-a0f0-ae7584de2306.png)

### Reproduction

https://stackblitz.com/edit/nuxt-starter-gdt6pi?file=pages/singlequote.vue
Does not work with Firefox (see #29) 
Click on "Single quote" and then on "OG:IMAGE" to see the image.

### System / Nuxt Info

```Shell
------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.3.2`
- Nitro Version:    `2.3.2`
- Package Manager:  `npm@7.17.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`
- Runtime Modules:  `nuxt-og-image@2.0.0-beta.4`
- Build Modules:    `-`
------------------------------
```

**Comment by harlan-zw** [comment]
Thanks! A fix is available in v2.0.0-beta.6

**Comment by harlan-zw** [comment]
Thanks! Looks like it's fixed in v2.0.0-beta.7
